### DISCUSSION TITLE
SCUM

### DISCUSSION THREAD

**Discussion by thomas93uk** [gameserverapp/Platform#215] [Game request]
SCUM: https://store.steampowered.com/app/513710/SCUM/

**Comment by mvaessen** [comment]
as of now, no dedicated server support
https://steamcommunity.com/app/513710/discussions/0/3825285417044115678/

**Comment by MichaelAW88** [comment]
when SCUM does allow it. it does need to be added

**Comment by mvaessen** [comment]
SCUM just released server files, and you can now find a blueprint on the marketplace: https://dash.gameserverapp.com/marketplace/market/blueprint/974
